### PR TITLE
HPCC-17502 Updating supported distributions

### PR DESCRIPTION
**Pull request by Michael-Gardner**
Signed-off-by: Michael Gardner <michael.gardner@lexisnexis.com>

### CODE REVIEW

**Comment by Michael-Gardner** [comment]
Rodrigo, please review

File: cmake_modules/dependencies/yakkety.cmake [hunk 1]
```diff
@@ -0,0 +1 @@
+set ( CPACK_DEBIAN_PACKAGE_DEPENDS  "hpccsystems-platform, ganglia-monitor, ganglia-monitor-python, python-lxml, rrdtool")
```
Comment (rpastrana) [inline cmake_modules/dependencies/yakkety.cmake:1]: is the ganglia-monitor dependent on "ganglia-monitor"?

File: cmake_modules/dependencies/zesty.cmake [hunk 2]
```diff
@@ -0,0 +1 @@
+set ( CPACK_DEBIAN_PACKAGE_DEPENDS  "hpccsystems-platform, ganglia-monitor, ganglia-monitor-python, python-lxml, rrdtool")
```
Comment (rpastrana) [inline cmake_modules/dependencies/zesty.cmake:1]: is the ganglia-monitor dependent on "ganglia-monitor"?

**Review by rpastrana — CHANGES_REQUESTED** [review]
@Michael-Gardner a couple of questions

File: cmake_modules/dependencies/yakkety.cmake (hunk 1 above)
Comment (Michael-Gardner) [reply cmake_modules/dependencies/yakkety.cmake:1]: http://packages.ubuntu.com/search?keywords=ganglia-monitor
ganglia-monitor is the actual ganglia package for ubuntu.  hpccsystems-ganglia-monitoring is the name of our package.

**Review by rpastrana — APPROVED** [review]
@Michael-Gardner it looks reasonable

**Comment by Michael-Gardner** [comment]
@richardkchapman Please merge

**Pull request merged**